java/src/jmri/jmrit/operations/locations/TrackTableModel.java

```diff
@@ -171,95 +201,96 @@ public int getColumnCount() {
-            case SCHEDULE_COLUMN:
-                return Bundle.getMessage("Schedule");
-            case RESTRICTION_COLUMN:
-                return Bundle.getMessage("Restrictions");
-            case LOAD_COLUMN:
-                return Bundle.getMessage("Load");
-            case SHIP_COLUMN:
-                return Bundle.getMessage("Ship");
-            case ROAD_COLUMN:
-                return Bundle.getMessage("Road");
-            case DESTINATION_COLUMN:
-                return Bundle.getMessage("Dest");
-            case POOL_COLUMN:
-                return Bundle.getMessage("Pool");
-            case PLANPICKUP_COLUMN:
-                return Bundle.getMessage("PlanPickUp");
-            case ALT_TRACK_COLUMN:
-                return Bundle.getMessage("AlternateTrack");
-            case ORDER_COLUMN:
-                return Bundle.getMessage("ServiceOrder");
-            case REPORTER_COLUMN:
-                return Bundle.getMessage("Reporters");
-            case EDIT_COLUMN:
-                return Bundle.getMessage("ButtonEdit"); // titles above all columns
-            default:
-                return "unknown"; // NOI18N
+        case ID_COLUMN:
+            return Bundle.getMessage("Id");
+        case NAME_COLUMN:
+            return Bundle.getMessage("TrackName");
+        case LENGTH_COLUMN:
+            return Bundle.getMessage("Length");
+        case USED_LENGTH_COLUMN:
+            return Bundle.getMessage("Used");
+        case RESERVED_COLUMN:
+            return Bundle.getMessage("Reserved");
+        case MOVES_COLUMN:
+            return Bundle.getMessage("Moves");
+        case LOCOS_COLUMN:
+            return Bundle.getMessage("Engines");
+        case CARS_COLUMN:
+            return Bundle.getMessage("Cars");
+        case PICKUPS_COLUMN:
+            return Bundle.getMessage("Pickups");
+        case SETOUT_COLUMN:
+            return Bundle.getMessage("Drop");
+        case SCHEDULE_COLUMN:
+            return Bundle.getMessage("Schedule");
+        case RESTRICTION_COLUMN:
+            return Bundle.getMessage("Restrictions");
+        case LOAD_COLUMN:
+            return Bundle.getMessage("Load");
+        case SHIP_COLUMN:
+            return Bundle.getMessage("Ship");
+        case ROAD_COLUMN:
+            return Bundle.getMessage("Road");
+        case DESTINATION_COLUMN:
+            return Bundle.getMessage("Dest");
+        case POOL_COLUMN:
+            return Bundle.getMessage("Pool");
+        case PLANPICKUP_COLUMN:
+            return Bundle.getMessage("PlanPickUp");
+        case ALT_TRACK_COLUMN:
+            return Bundle.getMessage("AlternateTrack");
+        case ORDER_COLUMN:
+            return Bundle.getMessage("ServiceOrder");
+        case REPORTER_COLUMN:
+            return Bundle.getMessage("Reporters");
+        case EDIT_COLUMN:
+            return Bundle.getMessage("ButtonEdit"); // titles above all columns
+        default:
+            return "unknown"; // NOI18N
```

Comment:
Is this reformatting done by purpose? Sometimes this can happen because some setting in the editor gets change by misstake.

If you want the formatting this way, I will not argue against it. But I don't think this is how JMRI usually format switch case statements.

Reply:
That's the way my IDE does it, but it also looks like the  standard format.   It's only a spacing issue with the reformatting.

Reply:
Now I see that the case statements are no longer indented.  Not sure why, in the past using the same IDE, it has been.  Something has changed, but not by me.

Reply:
Found and fixed the formatting issue with my IDE. Thanks for bring it to my attention.